### ISSUE TITLE
tinted images

### ISSUE THREAD

**Issue by sdevo619** [IvanVorobei/SPPermission#96]
Is there a way to use tinted images for the custom images on the permission vc. Been trying to find it in the documentation, but so far have only figured out how to set a custom image, but still not how to change the tint color. Is it possible to do this???

**Comment by IvanVorobei** [comment]
You need see tint color and rendering mode for your picture.

**Comment by sdevo619** [comment]
Thank you for the fast response, unfortunately this still does not answer my question. Where do I set the tint color in code, for the custom icons I am using in the permissions view? I currently have the icons rendering mode to set as template image, and the icons show, but they are tinted the wrong color, and I want to change the tint color. Thanks!!!

**Comment by IvanVorobei** [comment]
I think I understood your question. The `DataSource` protocol does not allow you to change the tint color, so you need to find a way to transfer the correct `UIImage`.

**Comment by sdevo619** [comment]
Got it. Thank you again for the fast reply, and for making SPPermissions. Not sure if there is a way to add those capabilities in a future release, but of if so, it would be awesome. I was able to make it work for now, as I just supplied non template images that were already an appropriate color. I began integrating this into one my project yesterday evening, and I was able to integrate into my project overnight and dramatically improve the flow of the initial startup of my app on first install. If you compare the two of them, the old version looks like a train wreck in comparison. Again, thank you for making this!!!!
